screens/HomeScreen/components/CustomText.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Text, StyleSheet, Platform } from 'react-native';
+
+const styles = StyleSheet.create({
+  title: {
+    fontSize: 18,
+    fontWeight: 'bold',
+    color: Platform.OS === 'ios' ? '#333' : '#000',
+  },
+  author: {
+    fontSize: 14,
+    fontWeight: Platform.OS === 'ios' ? '500' : 'bold',
+  }
+});
+
+const CustomText = ({ children, style, type }) => {
```

Comment:
+ good, but it'll better to create component <Typography /> and send into it settings via props, line type, text, customStyle....